workspaces/orchestrator/plugins/orchestrator/src/dataFormatters/WorkflowOverviewFormatter.ts

```diff
@@ -32,8 +33,18 @@ export interface FormattedWorkflowOverview {
   readonly category: string;
   readonly description: string;
   readonly format: WorkflowFormatDTO;
+  readonly availablity?: string;
 }
 
+export const available = 'Available';
```

Comment:
To keep the same code-style, can this be named:
```suggestion
export const AVAILABLE = 'Available';
```

And I keep up to your decision if you prefer to keep it in this file or moving to `constants.ts`

Reply:
ohhh now I understand what  "please move the strings into constants" means, which way you prefer?

Reply:
the constants.ts file :-)